cmd/openshift-install/main.go

```diff
@@ -102,4 +107,35 @@ func runRootCmd(cmd *cobra.Command, args []string) {
 	if err != nil {
 		logrus.Fatal(errors.Wrap(err, "invalid log-level"))
 	}
+
+	if version.CommitDate != "" {
+		logOldInstallerNotification(version.CommitDate)
+	}
+}
+
+func logOldInstallerNotification(commitDate string) {
+	commit, err := time.Parse("2006-01-02", commitDate)
+	if err != nil {
+		logrus.Warnf("Failed to parse commit date %q: %v", version.CommitDate, err)
+		return
+	}
+
+	releaseName, err := version.Version()
+	if err != nil {
+		logrus.Warnf("Failed to parse version: %v", err)
+		return
+	}
+
+	now := time.Now()
+	age := now.Sub(commit)
+	days := int(math.Floor(age.Hours() / 24))
+	arch := runtime.GOARCH
+	switch arch {
+	case "amd64":
+		arch = "x86_64"
+	}
+	uri := fmt.Sprintf("https://mirror.openshift.com/pub/openshift-v4/%s/clients/ocp/latest/", arch)
```

Comment:
It'd be ideal if we didn't push them over a minor version, so latest-X.Y

Reply:
Hmm, we should also figure out where to point folks who hit this code via OKD.  Maybe the full download URI should be passed in as a build-time variable, in the hopes that ART and CI build jobs can make a more targeted call?